source/mapping-rules/introduction.txt

```diff
@@ -105,6 +108,34 @@ Mapping rules can be defined with the following ``Migrate as`` options:
    * - :ref:`Migrate rows as Embedded documents <rm-embedded-documents>`
      - Translates each row as a set of fields or an embedded object in a child document identified by a foreign key relationship.
 
+Custom Fields and Filters
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can create mapping rules with :ref:`custom fields <rm-field-customizations>`. 
+Relational Migrator also supports :ref:`table filters <rm-table-filters>` to limit 
+data migrated from relational tables.
+
+Synthetic Foreign Keys
+----------------------
+
+You can create synthetic foreign keys in Relational Migrator to define a logical 
+relationship between your relational database tables. For more information,
```

Comment:
Optional - but you could consider writing this in the form "To (x), do (y)".

To define a logical relationship between... you can create synthetic...